src/UiPath.Workflow/Activities/RoslynExpressionValidator.cs

```diff
@@ -177,7 +177,11 @@ protected virtual IEnumerable<ValidationError> ProcessDiagnostics(ExpressionCont
         PrepValidation(expressionContainer);
 
         ModifyPreppedCompilationUnit(expressionContainer);
-        var diagnostics = expressionContainer.CompilationUnit.GetDiagnostics().Select(diagnostic =>
+        var diagnostics = expressionContainer
+            .CompilationUnit
+            .GetDiagnostics()
+            .Where(d=> d.Severity == DiagnosticSeverity.Error)
```

Comment:
I take it for designer we don't care about warnings.

Reply:
we don't really :)